### PR TITLE
refactor(rxFeedback): move Components/rxFeedback to Elements/Feedback

### PR DESCRIPTION
**Pull request by kevincareer427**
JIRA: https://jira.rax.io/browse/FRMW-623

### LGTMs
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

**Comment by kevincareer427** [comment]
## Before
before

## After
after

File: src/elements/Feedback/docs/examples/feedback.simple.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!-- Sample HTML goes here as a live example of how to the component can be used -->
```
Comment (CITguy) [inline src/elements/Feedback/docs/examples/feedback.simple.html]: Please remove this comment.

**Comment by kevincareer427** [comment]
@CITguy deleted.

File: src/elements/Feedback/scripts/rxFeedback.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
+angular.module('encore.ui.elements')
 /**
- * @ngdoc overview
- * @name rxFeedback
+ * @ngdoc directive
+ * @name elements.directive:rxFeedback
```
Comment (100stacks) [inline src/elements/Feedback/scripts/rxFeedback.js]: @kevincareer427, it looks like `src/components/rxFeedback/rxFeedback.module.js` was changed to `src/elements/Feedback/scripts/rxFeedback.js`.

In may not be an issue if you moved all the proper content.

Comment (CITguy) [inline src/elements/Feedback/scripts/rxFeedback.js]: I think this is a case of git optimizing the change. Most of the documentation in `rxFeedback.module.js` was appropriate for `rxFeedback.js` so it was moved. Git categorized it as a rename with modifications.

**Comment by CITguy** [comment]
LGTM

File: src/elements/Feedback/scripts/rxFeedback.js (hunk 2 above)
Comment (kevincareer427) [inline src/elements/Feedback/scripts/rxFeedback.js]: @100stacks Yep actually just the documentation was moved to rxFeedback.js, and rxFeedback .moduel.js was removed directly.

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**